### PR TITLE
User crud operations

### PR DESCRIPTION
**Pull request by onkar-josh**

### CODE REVIEW

File: mongodconn/mailApi.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const nodeMailer = require('nodemailer');
```
Comment (sunil4sonawane) [inline mongodconn/mailApi.js:1]: why this file is required here?

File: mongodconn/config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+function dbUrl(){
+const mongoDbUrl = 'mongodb://onkar:onkar@127.0.0.1:27017';
+return mongoDbUrl;
+}
+
+module.exports={
+	client,
+	dbUrl	
+}
```
Comment (sunil4sonawane) [inline mongodconn/config.js]: add 1 new blank line

File: mongodconn/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const routingFile= require("./userRouter.js");
+
+const file1 = require('./mongoConn.js');
```
Comment (sunil4sonawane) [inline mongodconn/index.js]: file1 is not meaningful

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
Please correct the mentioned issues

File: mongodconn/mailApi.js (hunk 1 above)
Comment (onkar-josh) [reply mongodconn/mailApi.js:1]: actually I have done this to notify user after inserting his data

**Pull request merged**